dapp_manager/storage.py

```diff
@@ -50,6 +49,14 @@ def read_file(self, file_type: RunnerFileType) -> str:
         except FileNotFoundError:
             return ""
 
+    def iter_file(self, file_type: RunnerFileType):  # ToDo type
+        try:
+            with open(self.file_name(file_type), "r") as f:
+                for l in f:
+                    yield l
+        except FileNotFoundError:
+            return
```

Comment:
I can see potential problems with when creating generator like this:
```python
foo = storage.iter_file()
```
instead of
```python
for line in storage.iter_file():
    ...
```